### ISSUE TITLE
still need this repo?

### ISSUE THREAD

**Issue by epageperron** [cdli-gh/parallel_texts#1]
@khoidt Do we still need this repo now that you have a corpus repo ?

**Comment by khoidt** [comment]
@epageperron Not at all. I plan to remove it after the corpus repo is populated (probably today).

**Comment by khoidt** [comment]
@epageperron this repo can be deleted, but it seems that I cannot do it. Can you try?
